scripts/create_baseline_stubs.py

```diff
@@ -169,11 +169,10 @@ def main() -> None:
 
     print("\nDone!\n\nSuggested next steps:")
     print(f" 1. Manually review the generated stubs in {stub_dir}")
-    print(f' 2. Run "MYPYPATH={stub_dir} python3 -m mypy.stubtest {package}" to check the stubs against runtime')
-    print(f' 3. Run "mypy {stub_dir}" to check for errors')
-    print(f' 4. Run "black {stub_dir}" and "isort {stub_dir}" (if you\'ve made code changes)')
-    print(f' 5. Run "flake8 {stub_dir}" to check for e.g. unused imports')
-    print(" 6. Commit the changes on a new branch and create a typeshed PR")
+    print(" 2. See CONTRIBUTING.md for all the best practices and tips to improve your stub")
+    print(' 3. Run autofixes and tests locally using "python3 -m ./scripts/run_all_checks.py"')
+    print("    See tests/README.md to run individual tests")
+    print(" 4. Commit the changes on a new branch and create a typeshed PR (don't force-push!)")
```

Comment:
I still don't think we should be telling contributors that they _have_ to run the tests prior to submitting a PR -- if they _want_ to run the tests first, that's fine, but I think it should also be absolutely fine if they just want to rely on CI :)

Also, I'd rather keep all our documentation on tests in one place -- would you mind adding a short description of the new script to `tests/README.md`?

```suggestion
    print(" 2. Optionally run tests and autofixes (see tests/README.md for details")
    print(" 3. Commit the changes on a new branch and create a typeshed PR (don't force-push!)")
```